akka-docs/src/main/paradox/scala/persistence.md

```diff
@@ -638,7 +638,7 @@ akka.persistence.journal.leveldb.replay-filter {
 <a id="snapshots"></a>
 ## Snapshots
 
-Snapshots can dramatically reduce recovery times of persistent actors.
+As you model your domain using actors, you may notice that some actors may be prone to accumulating extremely long event logs and experiencing long recovery times. Most often, the right approach is to split out into a set of shorter lived actors. However, when this is not an option, you can use snapshots to reduce recovery times drastically. 
```

Comment:
@johanandren I downgraded "most often" to "sometimes"... just curious, why would not splitting out be the right approach if your domain allows it (for example, splitting out a BankAccount into MonthlyBankAccount) ?